hbase-client/src/main/java/org/apache/hadoop/hbase/ServerMetricsBuilder.java

```diff
@@ -358,6 +360,8 @@ public String toString() {
       for (RegionMetrics r : getRegionMetrics().values()) {
         storeCount += r.getStoreCount();
         storeFileCount += r.getStoreFileCount();
+        storeRefCount += r.getStoreRefCount();
+        maxStoreFileRefCount += r.getMaxStoreFileRefCount();
```

Comment:
You should do max() here.

Reply:
This is addition of all counts for toString() right? I thought of doing max() but somehow I feel all these counts should be addition of all regionMetrics for the sake of toString() only